service_capacity_modeling/models/org/netflix/elasticsearch.py

```diff
@@ -82,6 +82,11 @@ def _estimate_elasticsearch_requirement(  # noqa: E501 pylint: disable=too-many-
     The input desires should be the **regional** desire, and this function will
     return the zonal capacity requirement
     """
+    # NOTE: ElasticSearch is only using buffers for disk but CPU and disk
```

Comment:
This should be a follow-up change. I noticed that there are hard coded buffers here and so I don't want to do it in this change to avoid a large blast radius.,